### ISSUE TITLE
how i can set md-primary style to md-nav-bar

### ISSUE THREAD

**Issue by devLaaziz** [angular/material#8654]
how i can set md-primary style to md-nav-bar

**Comment by DevVersion** [comment]
This is currently not possible, but would be a necessary enhancement.

**Comment by gmoothart** [comment]
Yeah, like md-tabs it takes the accent color. md-colors (slated for 1.1) will give you more freedom to specify the colors.

**Comment by seanlarge** [comment]
Hi I am trying to style the nav item with md-colors and it does not seem to work: <md-nav-item md-colors="{color: 'primary-50'}"

**Comment by jschleining** [comment]
The workaround I am creating right now is to create a second theme with the preferred palette set to accent. Then just apply the new theme to the md-nav-bar using md-theme. Note that this still uses the light colors in the palette, so you may have to adjust accordingly (I cheated horribly and created a new palette with all color presets set to the desired color). This is definitely not an ideal solution, and Im working on coming up with something better, but at least it is something for now that doesnt involve overwriting angular material's css

**Comment by jacobscarter** [comment]
+1 @seanlarge @gmoothart on the md-color issue for nav bar
